ui/src/app/cohort-review/participant-status/participant-status.component.spec.ts

```diff
@@ -0,0 +1,83 @@
+import {ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
+import {ReactiveFormsModule} from '@angular/forms';
+import {ActivatedRoute} from '@angular/router';
+import {Observable} from 'rxjs/Observable';
+
+import {ParticipantStatusComponent} from './participant-status.component';
+
+import {CohortReviewService, CohortStatus} from 'generated';
+import {ReviewStateService} from '../review-state.service';
+
+import {updateAndTick} from '../../../testing/test-helpers';
+import {Participant} from '../participant.model';
+
+class StubRoute {
+    snapshot = {
+        params: {
+            ns: 'workspaceNamespace',
+            wsid: 'workspaceId',
+            cid: 1
+        },
+        data: {
+            workspace: {cdrVersionId: 1}
+        }
+    };
+}
+
+class ApiSpy {
+    updateParticipantCohortStatus = jasmine
+        .createSpy('updateParticipantCohortStatus')
+        .and
+        .returnValue(
```

Comment:
NIT: put this and the next two lines on one, please.  You should have enough column space (you definitely will after the indent size is fixed).  If its still too long break the observable out into its own assignment.

```
const participant$ = Observable.of(/* etc */);

/*  later  */
  .returnValue(participant$);
```

Reply:
Change indent to 2 spaces and reformatted file. Done.